docs/src/modules/components/AdInHouse.js

```diff
@@ -52,7 +52,7 @@ export default function AdInHouse(props) {
         className={classes.link}
         href={ad.link}
         target="_blank"
-        rel="noopener sponsored"
+        rel="noopener noreferrer sponsored"
```

Comment:
Don't we want the referrer for analytics tracking?

Reply:
Yeah, sounds fair, reverting.